### PR TITLE
Em okta 675369 update cypress

### PR DESCRIPTION
**Pull request by emikhasyak-okta**
## changes
- updates cypress package and migrates tests
- updates node for CCI and Bacon

I've checked yarn.lock after the fix and it has correct packages mentioned by dependabot https://github.com/okta/okta-help/security/dependabot

Bacon: https://bacon-go.aue1e.saasure.net/commits?artifact=okta-help&branch=em-OKTA-675369-update-cypress&page=1&pageSize=6&sha=d496e946a4f3e018e4a5ad4749b0125f7af45a6f&tab=main
CCI: https://app.circleci.com/pipelines/github/okta/okta-help?branch=em-OKTA-675369-update-cypress
image

## Jira
- [OKTA-675369](https://oktainc.atlassian.net/browse/OKTA-675369)

## Reviewer
- @paulwallace-okta

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  screenshotsFolder: 'tests/cypress/screenshots',
+  videosFolder: 'tests/cypress/videos',
+  fixturesFolder: 'tests/cypress/fixtures',
```
Comment (paulwallace-okta) [inline cypress.config.js:6]: I suspect this file was autogenerated. 
It's missing the entries for `integrationFolder` and `supportFolder`, as in the deleted JSON file.
When I try to run this locally, Cypress automatically creates a top-level `/cypress` folder, with default integration tests.
I quit Cypress, removed this new folder, and added the missing config data to this file, this fixes everything.

Comment (emikhasyak-okta) [reply cypress.config.js:6]: Thanks for catching this! I did run tests in all environments but haven't looked carefully at the output. False positives are the worst.

Comment (emikhasyak-okta) [reply cypress.config.js:6]: hmm, have you ran `yarn` command? If you run old cypress on new thing it should not work.
I can see tests passing local (the actually tests)
Logs in bacon also are showing tests: 
image

and the same for CircleCI

Comment (paulwallace-okta) [reply cypress.config.js:6]: I pulled down the branch, then ran `yarn run cy:open`, as I usually do.

Comment (emikhasyak-okta) [reply cypress.config.js:6]: In that case, the "old" cypress is not working well with migrated tests. If you run `yarn` to fetch the correct cypress that should fix it.

Comment (paulwallace-okta) [reply cypress.config.js:6]: and, the `yarn run cy:run` command just fails, can't find the spec files

**Review by paulwallace-okta — APPROVED** [review]
🚀

**Pull request merged**